control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go

```diff
@@ -261,6 +261,12 @@ func ReconcileDeployment(dep *appsv1.Deployment, params Params, apiPort *int32)
 	if params.ExposedThroughHCPRouter {
 		cnoEnv = append(cnoEnv, corev1.EnvVar{Name: "OVN_SBDB_ROUTE_LABELS", Value: util.HCPRouteLabel + "=" + dep.Namespace})
 	}
+	if params.IsPrivate {
+		cnoEnv = append(cnoEnv, corev1.EnvVar{Name: "OVN_SBDB_ROUTE_LABELS", Value: fmt.Sprintf("%v=%v,%v=%v",
```

Comment:
This would result in the env var getting added twice, because 'params.ExposedThroughHCPRouter' includes the private case as well.